matrix-meetings-bot/test/MeetingService.test.ts

```diff
@@ -1804,6 +1804,52 @@ describe('test relevant functionality of MeetingService', () => {
         callInfo(0, 3, StateEventName.IO_ELEMENT_WIDGETS_LAYOUT_EVENT)
       ).toStrictEqual(expected);
     });
+
+    test('update room layout when widgets change', async () => {
+      const event = createEvent(parentId);
+      let widgets = [WidgetType.COCKPIT, WidgetType.BREAKOUT_SESSIONS, 'poll'];
+      event.widget_ids = widgets;
+      await meetingService.createMeeting(userContext, event);
+
+      // should not have custom layout
+      verify(
+        clientMock.sendStateEvent(
+          parentId,
+          StateEventName.IO_ELEMENT_WIDGETS_LAYOUT_EVENT,
+          anything(),
+          anything()
+        )
+      ).times(0);
+
+      // add jitsi widget
+      widgets = ['jitsi'];
+      await meetingService.handleWidgets(
+        userContext,
+        new MeetingWidgetsHandleDto(parentId, true, widgets)
+      );
+
+      // the resulting layout should exactly match the custom configuration
+      const layout = layoutConfigs.find((o) =>
+        _.isEqual(_.sortBy(o.widgetIds), _.sortBy(widgets))
+      );
+      const expected = {
+        widgets: layout?.layouts,
+      };
```

Comment:
It loads config for `'jitsi'`, not for `'poll'` and `'jitsi'`. I suggest to copy the needed config into the test directly to see what value really is expected here.